.run/Run cluster command.run.xml

```diff
@@ -0,0 +1,24 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run cluster command" type="GradleRunConfiguration" factoryName="Gradle">
+    <ExternalSystemSettings>
+      <option name="executionName" />
+      <option name="externalProjectPath" value="$PROJECT_DIR$" />
+      <option name="externalSystemIdString" value="GRADLE" />
+      <option name="scriptParameters" value="" />
+      <option name="taskDescriptions">
+        <list />
+      </option>
+      <option name="taskNames">
+        <list>
+          <option value="run" />
+          <option value="--args=&quot;cluster &lt;/path/to/submissions.csv&gt; &lt;distance limit&gt; --outputDir=&lt;/path/to/output/dir&gt; --language=PYTHON --saveClusters --clusteringResult&quot;" />
```

Comment:
Should we put default distance limi here?

Reply:
Set 60 as default distance limit in run configuration, thanks!